### PR TITLE
fixed generate_tests.py and added new generated generated.yaml

### PR DESCRIPTION
**Pull request by asim-reza**
### Issue
[error unmarshaling config/jobs/kubernetes/generated/generated.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"resources\" #27430](https://github.com/kubernetes/test-infra/issues/27430)

as mentioned in the aforementioned issue

### CODE REVIEW

**Comment by asim-reza** [comment]
@pohly this is a similar PR to PR #27477 do you mind checking this out when you do that... cheers

File: config/jobs/kubernetes/generated/generated.yaml
Comment (pohly) [inline config/jobs/kubernetes/generated/generated.yaml:297]: Should this `resources: *id001` still be here?

Comment (asim-reza) [reply config/jobs/kubernetes/generated/generated.yaml:297]: Yes... as resources still need to be added here but with the correct indentation now.... the script abriviates the useage

Comment (asim-reza) [reply config/jobs/kubernetes/generated/generated.yaml:297]: I believe `resources: *id001` was being added with wrong indentation before (2 lines below)

Comment (asim-reza) [reply config/jobs/kubernetes/generated/generated.yaml:297]: Yes... as resources still need to be added here but with the correct indentation now.... the script abriviates the useage...I believe resources: *id001 was being added with wrong indentation before (2 lines below)

Comment (asim-reza) [reply config/jobs/kubernetes/generated/generated.yaml:297]: resources: *id001 in the original script was intended to replace the template values but it got appended due to the bug in the script

Comment (pohly) [reply config/jobs/kubernetes/generated/generated.yaml:297]: Yes, but now it is empty. If it needs to be empty, then we don't need the `resources: *id001` line. But I suspect that it is meant to have some valid content.

Comment (asim-reza) [reply config/jobs/kubernetes/generated/generated.yaml:297]: it points to and populates from
```resources: &id001
        requests:
          cpu: 1000m
          memory: 3Gi
          cpu: 2000m
          memory: 6Gi```

Comment (asim-reza) [reply config/jobs/kubernetes/generated/generated.yaml:297]: its short hand code using an anchor(&) and and alias (*)

Comment (pohly) [reply config/jobs/kubernetes/generated/generated.yaml:297]: I learned something new today - I wasn't aware that YAML can do that.

But I still find it odd that a generated file uses this mechanism. I'll defer to the owner of this particular job whether they want this.

Comment (asim-reza) [reply config/jobs/kubernetes/generated/generated.yaml:297]: where does it list specific job owners?

**Review by asim-reza — COMMENTED** [review]
need someone to check if the changes make logical sense for the functionality of the affected jobs

**Comment by dims** [comment]
@kubernetes/sig-release-leads please review!

**Comment by asim-reza** [comment]
@palnabarun rebase didnt work nicely (Im still not great at GitOps) as there was a conflict so I merged master and resolved the conflict

**Comment by asim-reza** [comment]
Note: dont worry about `pull-test-infra-prow-checkconfig-beta` this should resolve after this and another PR (currently in the merge pool) merge

**Review by palnabarun — APPROVED** [review]
/lgtm
/approve

**Comment by palnabarun** [comment]
@asim-reza -- just for documentation, can you also link the PR here which resolves the test failure?

**Comment by asim-reza** [comment]
> @asim-reza -- just for documentation, can you also link the PR here which resolves the test failure?

#27459 
#27460 
#27462 
#27466 
#27467 
#27469 
#27474 
#27478 
#27487 
#27494 
#27495 
#27513 
#27514 
#27515 
#27516 
#27526 
#27529 

the one that was in the merge pool was: #27494

**Comment by palnabarun** [comment]
Thank you!

**Pull request merged**
